Assets/BossRoom/Scripts/Server/ServerTestingHotkeys.cs

```diff
@@ -58,6 +65,13 @@ private void Update()
             {
                 NetworkManager.SceneManager.LoadScene("PostGame", LoadSceneMode.Single);
             }
+            if (m_GodModeKeyCode != KeyCode.None && Input.GetKeyDown(m_GodModeKeyCode))
+            {
+                if (m_BossRoomState)
+                {
+                    m_BossRoomState.ToggleGodMode();
+                }
+            }
```

Comment:
Another thing which I thought of after reviewing: this new godMode cheat deserves some UI display, since anyone toggling the G key accidentally will now think the game is unresponsive.

Reply:
You're right. At the moment I only added a Console.Log and even knowing about it I feel it can be easy to miss.
There is [this ticket](https://jira.unity3d.com/browse/MTT-1994) about adding a popup window for cheats. I think it would be a good solution. Or if we want something specific that is always visible, we could add a label similar to the NetworkLatencyWarning